src/nbl/video/ILogicalDevice.cpp

```diff
@@ -291,92 +292,87 @@ core::smart_refctd_ptr<IGPUShader> ILogicalDevice::createShader(const SShaderCre
     // https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VkPipelineShaderStageCreateInfo.html#VUID-VkPipelineShaderStageCreateInfo-stage-00706
     switch (shaderStage)
     {
-        case IGPUShader::E_SHADER_STAGE::ESS_VERTEX:
-            break;
-        case IGPUShader::E_SHADER_STAGE::ESS_TESSELLATION_CONTROL: [[fallthrough]];
-        case IGPUShader::E_SHADER_STAGE::ESS_TESSELLATION_EVALUATION:
-            if (!features.tessellationShader)
-            {
-                NBL_LOG_ERROR("Cannot create IGPUShader for %p, Tessellation Shader feature not enabled!", creationParams.cpushader);
-                return nullptr;
-            }
-            break;
-        case IGPUShader::E_SHADER_STAGE::ESS_GEOMETRY:
-            if (!features.geometryShader)
-            {
-                NBL_LOG_ERROR("Cannot create IGPUShader for %p, Geometry Shader feature not enabled!", creationParams.cpushader);
-                return nullptr;
-            }
-            break;
-        case IGPUShader::E_SHADER_STAGE::ESS_FRAGMENT: [[fallthrough]];
-        case IGPUShader::E_SHADER_STAGE::ESS_COMPUTE:
-            break;
-        // unsupported yet
-        case IGPUShader::E_SHADER_STAGE::ESS_TASK: [[fallthrough]];
-        case IGPUShader::E_SHADER_STAGE::ESS_MESH:
-            NBL_LOG_ERROR("Unsupported (yet) shader stage");
+    case IGPUShader::E_SHADER_STAGE::ESS_TESSELLATION_CONTROL: [[fallthrough]];
+    case IGPUShader::E_SHADER_STAGE::ESS_TESSELLATION_EVALUATION:
```

Comment:
dont change my indent, I like my cases indented one more than `switch`

Reply:
VS automatically did it..